Validation/EcalDigis/src/EcalSelectiveReadoutValidation.cc

```diff
@@ -1378,7 +1378,13 @@ EcalSelectiveReadoutValidation::analyzeTP(edm::Event const & event,
     double etSum = ttEtSums[iEta0][iPhi0];
 
     int iE = meTp_->getTProfile()->FindFixBin(tpEt);
-    ++tpEtCount[iE];
+    if ((iE >= 0) && (iE < 100)) {
+      ++tpEtCount[iE];
+    } else {
+      // FindFixBin might return an overflow bin (outside tpEtCount range).
+      // To prevent a memory overflow / segfault, these values are ignored.
+      std::cout << "EcalSelectiveReadoutValidation: Invalid iE value: " << iE << std::endl;
```

Comment:
I've copied the std::cout code from a different location in the same file.
Point was to fix the error with least changes to the code and the code style.

I agree that we shouldn't use std::cout, but currently it's all over the code.
I will discuss it further and make a separate pull request(s) converting those.